pages/common/transmission-daemon.md

```diff
@@ -0,0 +1,21 @@
+# transmission-daemon
+
+> Daemon controlled with `transmission-remote` or its web interface.
+> See also: `transmission`.
+> More information: <https://manned.org/transmission-daemon>.
+
+- Start a headless Transmission session:
```

Comment:
```suggestion
- Start a headless `Transmission` session:
```

Reply:
I guess since `Transmission` isn't actually a command to run anything, I was just treating it as more of its name. If you think this is an important distinction, then maybe it should just be this?
```suggestion
- Start a headless `transmission` session:
```

Reply:
Yes, I agree. I don't think it should be capitalized.